data.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/lib/pq"
+)
+
+// lookup table for buildings that we have in our system.
+var parentNameLookup = map[int]string{
+	146: "Avery",
+	103: "Butler",
+	62:  "East Asian Library",
+	75:  "John Jay",
+	79:  "Lehman Library",
+	84:  "Lerner",
+	15:  "Northwest Corner Building",
+	2:   "Uris",
+}
+
+// dataset is an alias for an array of data dumps
+type dataset []dumpFormat
+
+// dumpFormat represents the datapoints provided for the wireless data.
+// Some fields are gathered from the dumped JSON files and others are configured
+// afterwards before storage.
```

Comment:
Yeah you don't need to comment on the lines directly, but maybe be explicit in saying "`GroupName`, `ParentID`, and `ClientCount` are automatically populated, while `ParentName`, `GroupID`, and `DumpTime` must be generated manually.